### DISCUSSION TITLE
浅谈 JavaScript 异步编程：前言

### DISCUSSION THREAD

**Discussion by giscus** [darkyzhou/blog-comments#2] [General]
# 浅谈 JavaScript 异步编程：前言

 2009 年，Ryan Dahl 正式发布服务端 JavaScript 解释器 Node.js，以其极高的性能震惊世人。在发布后的短短几年之间，Node.js 攻城略池，在 Web 服务端领域中占据了相当大的市场份额，并在之后的几年为伟大的前后端分离策略的诞生提供了物质基础。
Node.js 的成功也让 JavaSc...

https://darkyzhou.net/articles/async-js

**Comment by tmKnight01** [comment]
runtime和js引擎角逐那块真是让我醍醐灌顶，受教了!
